conda.recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - python
     - anaconda-anon-usage >=0.4.0
     - conda-package-handling >=1.7.3
+    - conda-package-streaming
```

Comment:
Is there a lower-bound we should specify here?

Reply:
0.9.0 is a decent minimum https://github.com/conda/conda-package-handling/blob/main/conda.recipe/meta.yaml#L29C1-L29C38